### PR TITLE
ADDED issue template for bug report

### PR DESCRIPTION
**Pull request by Caketime**
ADDED basic code of conduct
image
image
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
```
Comment (juliusmarminge) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:4]: unconfirmed_bug is probably more used and then we switch to bug once its reproduced

Thats how we do in t3-oss and trpc atleast

**Comment by Caketime** [comment]
Force pushed an update so it stays a single commit.

Changes: Label "bug" -> "uncofirmed_bug"

**Review by markflorkowski — CHANGES_REQUESTED** [review]
I'd also like to have "Reproduction Repository" and "Reproduction Steps" fields. They can be optional. 

Otherwise looks good!

**Comment by Caketime** [comment]
- Added optional "Reproduction Repository" and "Reproduction Steps" fields

- Updated the "what did you expect to happen" description from "What did you expect to happen?" to "Please describe what you expected to happen."

**Review by markflorkowski — APPROVED** [review]

**Pull request merged**